source/frontend/src/features/mapSideBar/acquisition/tabs/compensation/detail/CompensationRequisitionDetailView.test.tsx

```diff
@@ -31,7 +31,9 @@ describe('Compensation Detail View Component', () => {
       <CompensationRequisitionDetailView
         acquisitionFile={renderOptions?.props?.acquisitionFile ?? mockAcquisitionFileResponse()}
         compensation={renderOptions?.props?.compensation ?? getMockApiDefaultCompensation()}
-        compensationProperties={renderOptions?.props?.compensationProperties ?? getMockCompensationPropertiesReq()}
+        compensationProperties={
```

Comment:
?

Reply:
I think this is just a lint thing. I no longer see this file on my PR update.